ChiModules/LinearBoltzmannSolver/lbs_04c_io_flux_moments.cc

```diff
@@ -0,0 +1,326 @@
+#include "lbs_linear_boltzmann_solver.h"
+
+#include "chi_log.h"
+extern ChiLog& chi_log;
+
+#include "chi_mpi.h"
+extern ChiMPI& chi_mpi;
+
+#include <fstream>
+#include <cstring>
+
+//###################################################################
+/**Makes a source-moments vector from scattering and fission based
+ * on the latest phi-solution.*/
+void LinearBoltzmann::Solver::
+  MakeSourceMomentsFromPhi(std::vector<double> &source_moments)
```

Comment:
I would suggest having this method return the source moments vector instead of taking as an argument (rvo should make it just as fast).

Reply:
Done